### PR TITLE
Spaceport news stories for Hai

### PR DESCRIPTION
**Pull request by NomadicVolcano**
These were made for #4943 as a basis of the lore there, but these spaceport news stories are good enough to stand on their own, so I've made a PR for them alone.

NOTE: Out of necessity (to set up lore) this PR is already merged into #4943, so if that PR is merged, this one has been as well.

- **Merchant captain**, "I met some young Hai pilots. It sounds like they start out as poor as we do."
- **Reformed pirate**, "I used to rob people for a living, and I'd be in jail now if I stayed in human space. Here, they've given me a job, and I've been sober for six weeks. This has really turned my life around."
- **Tourist**, "I saw the Hai-Human fusion band Nuclear Martini playing on Luna. Nobody realized the hairy drummer was really a Hai in disguise."
- **Scientist or engineer**, "A while back, my Hai research partner was nearing a breakthrough, and then she vanished. I haven't seen her since. I wonder where she went?"
- **Merchant captain**, "Some Hai were protesting the work week. They want to work only 32 hours a week instead of 40. I used to work 70 hours a week for the Syndicate!"
- **Visiting chef**, "A Hai-Human fusion restaurant near here sells peanut pizza. It sounds gross, but the taste is heavenly."

### CODE REVIEW

**Comment by NomadicVolcano** [comment]
Huggable

**Comment by NomadicVolcano** [comment]
Stew

**Comment by NomadicVolcano** [comment]
Cruise

**Comment by NomadicVolcano** [comment]
Pet

I swear this is the last one I'll post.

**Comment by downRightCoder256** [comment]
maybe have the reformed pirate have some of the less disconcerting portraits, and even some merchant portraits? the are reformed pirates, afterall.

**Comment by NomadicVolcano** [comment]
> Maybe have the reformed pirate have some of the less disconcerting portraits, and even some merchant portraits? They are reformed pirates, afterall.

He's wearing a mask because his face is mauled. Turning his life around doesn't fix that.

**Comment by downRightCoder256** [comment]
now thats a lore bit i wasn't aware of

**Comment by NomadicVolcano** [comment]
> now thats a lore bit i wasn't aware of

I was joking. I just updated the reformed pirate portraits.

**Comment by downRightCoder256** [comment]
is checking usually this slow ?

**Comment by NomadicVolcano** [comment]
> is checking usually this slow ?

The MacOS checks have to download and upload a lot of stuff. It slows them down.

**Comment by downRightCoder256** [comment]
Oh well, I guess its a necessary evil, then

**Comment by NomadicVolcano** [comment]
I'm starting to wonder whether it makes sense to split this out from #4943 since about a third of the new news stories serve to add the lore required by that PR. If anyone has opinions on that, please let me know.

**Comment by beccabunny** [comment]
On one hand, both are milestones for 0.9.13, and contain related content; on the other hand, the content of this PR is standalone enough from #4943. I'd say it's up to you, since imo they can stay separate but if you see that keeping both PRs up to date with each other is too much of a hassle you should merge them together

**Comment by NomadicVolcano** [comment]
> On one hand, both are milestones for 0.9.13, and contain related content; on the other hand, the content of this PR is standalone enough from #4943. I'd say it's up to you, since imo they can stay separate but if you see that keeping both PRs up to date with each other is too much of a hassle you should merge them together

I can type "git merge master" pretty quickly. It's more about whether it would be easier for Amazinite to merge it if I separate them or not.

File: data/hai/hai news.txt [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+			"Tour guide"
+	portrait
+		portrait/human26
+		portrait/human31
+		portrait/human57
+		portrait/human68
+	message
+		word
+			"This may sound odd, but the acorn cookies they sell here are the best cookies I've ever eaten."
+			"A Hai-Human fusion restaurant near here sells peanut pizza. It sounds gross, but the taste is heavenly."
```
Comment (Amazinite) [inline data/hai/hai news.txt:30]:
```suggestion
			"A Hai-human fusion restaurant near here sells peanut pizza. It sounds gross, but the taste is heavenly."
```

File: data/hai/hai news.txt [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+			"Food connoisseur"
+			"Tourist"
+			"Tour guide"
+	portrait
+		portrait/human26
+		portrait/human31
+		portrait/human57
+		portrait/human68
+	message
+		word
```
Comment (Amazinite) [inline data/hai/hai news.txt:28]: Might want to add a message about the pickled acorns mentioned in the Centipede's description.

File: data/hai/hai news.txt [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+		portrait/human76
+		portrait/human79
+		portrait/human98
+		portrait/human108
+		portrait/human112
+		portrait/human125
+	message
+		word
+			"Hai Tech is incredible compared to ours. Every time I add a ship to my fleet, I always make a trip up here to stock up on Hai tech."
+			"Have you tried out one of those Hai anti-missile systems? They'll waste any missile in Human space."
```
Comment (Amazinite) [inline data/hai/hai news.txt:64]:
```suggestion
			"Have you tried out one of those Hai anti-missile systems? They'll waste any missile in human space."
```

File: data/hai/hai news.txt [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+			"Can you believe these Hai? They're light-years ahead of us, right under our noses."
+			"Have you visited the Quarg ringworld here in Hai space? It's a humbling sight. There's probably more Quarg in there than there are humans anywhere. Makes you feel pretty insignificant."
+			"Did you see the stuff they sell on Icelake? Hai technology is amazing."
+			"I met some young Hai pilots. It sounds like they start out as poor as we do."
+			"I sold my entire stock of food at this port. I guess even Hai get tired of nuts."
```
Comment (Amazinite) [inline data/hai/hai news.txt:69]: The implication here is that it's non-Hai food, so maybe the merchant should explicitly state that.
```suggestion
			"I sold my entire stock of food from human space at this port. I guess even Hai get tired of nuts."
```

File: data/hai/hai news.txt [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+	portrait
+		portrait/human13
+		portrait/human37
+		portrait/human43
+		portrait/human64
+		portrait/human71
+		portrait/human83
+	message
+		word
+			"A while back, my Hai research partner was nearing a breakthrough, and then she vanished. I haven't seen her since. She probably joined the Unfettered."
```
Comment (Amazinite) [inline data/hai/hai news.txt:90]: Do we really want to outright state "She probably joined the Unfettered?" This idea seems like one of those things that the news should only hint at, not explicitly state, with spaceport mission being where we really tackle the idea.

File: data/hai/hai news.txt [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+			"Can you believe these Hai? They're light-years ahead of us, right under our noses."
+			"Have you visited the Quarg ringworld here in Hai space? It's a humbling sight. There's probably more Quarg in there than there are humans anywhere. Makes you feel pretty insignificant."
+			"It's amazing how effective Hai fertilizer has become. If we had that technology, hunger would be a thing of the past."
+			"Hai can terraform planets a hundred times faster than we can. I wonder why they didn't do that at Darkwaste?"
+			"Hai have the most amazing holographic technology. One of them put on a backpack holoprojector. When he turned it on, he looked like an elephant. The legs and tail even tracked his movements! I wonder if they can mimic humans too?"
+			"The Hai have an amazing grasp of quantum chromodynamics. It's why their shield generators are so effective."
+			"The alloys Hai use in their ships are way beyond our technology."
```
Comment (Amazinite) [inline data/hai/hai news.txt:97]: This blurb feels pretty short compared to the rest. Maybe add a bit extra?

File: data/hai/hai news.txt [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+		portrait/human69
+		portrait/human107
+		portrait/human114
+		portrait/human123
+		portrait/human128
+	message
+		word
+			"My little daughter loved her stuffed bear back in human space, but now she wants to hug every Hai she sees instead."
+			"These Hai are so fluffy I want to hug one every time I see them!"
+			"I saw the Hai-Human fusion band Nuclear Martini playing on Luna. Nobody realized the hairy drummer was really a Hai in disguise."
```
Comment (Amazinite) [inline data/hai/hai news.txt:124]:
```suggestion
			"I saw the Hai-human fusion band Nuclear Martini playing on Luna. Nobody realized the hairy drummer was really a Hai in disguise."
```

File: data/hai/hai news.txt [hunk 8]
```diff
@@ -0,0 +1,251 @@
...
+		portrait/human79
+		portrait/human98
+		portrait/human108
+		portrait/human112
+		portrait/human125
+	message
+		word
+			"Don't go too far north up here. The Hai have their own pirates, and they're much more powerful than anything we're used to in human space."
+			"I took a shortcut through Wah Ki with six ships. Just two seconds later, BAM! half my fleet is dust."
+			"I hear Hai space wasn't peaceful a few centuries ago when some reptiles were raiding the place. I wonder where they went?"
```
Comment (Amazinite) [inline data/hai/hai news.txt:157]:
```suggestion
			"I hear Hai space wasn't peaceful a few centuries ago when some reptilian aliens were raiding the place. I wonder where they went?"
```

File: data/hai/hai news.txt [hunk 9]
```diff
@@ -0,0 +1,251 @@
...
+		portrait/human66
+		portrait/human69
+		portrait/human107
+		portrait/human114
+		portrait/human123
+		portrait/human128
+	message
+		word
+			"My little daughter loved her stuffed bear back in human space, but now she wants to hug every Hai she sees instead."
+			"These Hai are so fluffy I want to hug one every time I see them!"
```
Comment (Amazinite) [inline data/hai/hai news.txt:123]: This reads like a run-on sentence to me. A period, comma, or exclamation point would do to fix that.
```suggestion
			"These Hai are so fluffy! I want to hug one every time I see them!"
```

File: data/hai/hai news.txt [hunk 10]
```diff
@@ -0,0 +1,251 @@
...
+		portrait/human112
+		portrait/human125
+	message
+		word
+			"Don't go too far north up here. The Hai have their own pirates, and they're much more powerful than anything we're used to in human space."
+			"I took a shortcut through Wah Ki with six ships. Just two seconds later, BAM! half my fleet is dust."
+			"I hear Hai space wasn't peaceful a few centuries ago when some reptiles were raiding the place. I wonder where they went?"
+			"These Hai are so creepy. What are they hiding? I can't wait to get back to human space."
+			"The Hai don't like my ship's armory of laser rifles, but I was once boarded by a pirate crew and I'm never letting that happen again! If some overgrown squirrel is going to take away my rifle, it'll have to be from my cold dead hands!"
+			"I met some young Hai pilots. It sounds like they start out as poor as we do."
```
Comment (Amazinite) [inline data/hai/hai news.txt:160]: This one is already under a section with the same portraits and name. Might as well remove it.

File: data/hai/hai news.txt [hunk 11]
```diff
@@ -0,0 +1,251 @@
...
+		portrait/human76
+		portrait/human79
+		portrait/human98
+		portrait/human108
+		portrait/human112
+		portrait/human125
+	message
+		word
+			"Don't go too far north up here. The Hai have their own pirates, and they're much more powerful than anything we're used to in human space."
+			"I hear Hai space wasn't peaceful a few centuries ago when some reptiles were raiding the place. I wonder where they went?"
```
Comment (Amazinite) [inline data/hai/hai news.txt:192]:
```suggestion
			"I hear Hai space wasn't peaceful a few centuries ago when some reptilian aliens were raiding the place. I wonder where they went?"
```

File: data/hai/hai news.txt [hunk 12]
```diff
@@ -0,0 +1,251 @@
...
+			"My Hai friend went on one of those prison cruises a while back, but it got lost in an ion storm near Allhome."
```
Comment (Amazinite) [inline data/hai/hai news.txt:193]: What in the world is an ion storm? (We don't have system hazards yet.)

File: data/hai/hai news.txt [hunk 13]
```diff
@@ -0,0 +1,251 @@
...
+			"One of my Hai co-workers went to Sunracer to formalize a business deal. I warned her that part of human space is dangerous. I hope I'll see her again."
```
Comment (Amazinite) [inline data/hai/hai news.txt:194]: I'd delete this one. If any Hai are going to human space, it's not to make business deals.

File: data/hai/hai news.txt [hunk 14]
```diff
@@ -0,0 +1,251 @@
...
+			"A Hai that lives near me went to sleep one night and woke up in a box on Prime. Isn't that weird?"
```
Comment (Amazinite) [inline data/hai/hai news.txt:195]: What?

File: data/hai/hai news.txt [hunk 15]
```diff
@@ -0,0 +1,251 @@
...
+			"I always said I'd get out of piracy one day, but I know the navy would lock me up for sure. Here in Hai space, I've made a new start."
+			"Being in Hai space has taught me to be a better person."
+			"Some of my friends abuse the Hai's kindness. After all they've done for me, I'd never do that."
+			"Back home, I had to rob people just to get enough money for food. Here I never go hungry."
+			"Don't judge me. You have no idea what I had to go through before I got here."
+			"I haven't stabbed anyone in months. This is an easier life."
+			"Hai are so forgiving here. I got into a fight with one and instead of pressing charges, he had a long talk with me about my choices in life."
+			"I always said I'd rethink my life, and now I have. I'm a farmer, not a killer."
+			"I used to hate myself for all the terrible crimes I did in human space. Here, the Hai have given me a fresh start."
+			"You have to re-invent yourself sometimes. I used to be a killer on a pirate ship, now I'm a canner in Hai factory."
```
Comment (Amazinite) [inline data/hai/hai news.txt:245]:
```suggestion
			"You have to re-invent yourself sometimes. I used to be a killer on a pirate ship, now I'm a canner in a Hai factory."
```

File: data/hai/hai news.txt [hunk 16]
```diff
@@ -0,0 +1,251 @@
...
+			# Duplicated intentionally so pirates say it too:
+			"I hope nobody is smuggling Hai technology out of here. I don't want to think about what would happen if the pirates got their hands on that stuff."
+			"A Hai that lives near me went to sleep one night and woke up in a box on Prime. Isn't that weird?"
+			"Hai scientists that live near me chartered a flight to Allhome, but it got lost and ended up all the way on Canyon. Some people have the worst sense of direction."
+			"These Hai are so fluffy I want to hug one every time I see them!"
```
Comment (Amazinite) [inline data/hai/hai news.txt:250]:
```suggestion
			"These Hai are so fluffy! I want to hug one every time I see them!"
```

File: data/hai/hai news.txt [hunk 17]
```diff
@@ -0,0 +1,251 @@
...
+			"Back home, I had to rob people just to get enough money for food. Here I never go hungry."
+			"Don't judge me. You have no idea what I had to go through before I got here."
+			"I haven't stabbed anyone in months. This is an easier life."
+			"Hai are so forgiving here. I got into a fight with one and instead of pressing charges, he had a long talk with me about my choices in life."
+			"I always said I'd rethink my life, and now I have. I'm a farmer, not a killer."
+			"I used to hate myself for all the terrible crimes I did in human space. Here, the Hai have given me a fresh start."
+			"You have to re-invent yourself sometimes. I used to be a killer on a pirate ship, now I'm a canner in Hai factory."
+			# Duplicated intentionally so pirates say it too:
+			"I hope nobody is smuggling Hai technology out of here. I don't want to think about what would happen if the pirates got their hands on that stuff."
+			"A Hai that lives near me went to sleep one night and woke up in a box on Prime. Isn't that weird?"
```
Comment (Amazinite) [inline data/hai/hai news.txt:248]: Still what?

File: data/hai/hai news.txt (hunk 5 above)
Comment (NomadicVolcano) [reply data/hai/hai news.txt]: She didn't join the unfettered. She was kidnapped by the Devil-Run pirates. The purpose of this and some other new spaceport news is to set up the lore for the Devil-Run kidnappings. I removed the reference to the unfettered.

File: data/hai/hai news.txt (hunk 6 above)
Comment (NomadicVolcano) [reply data/hai/hai news.txt]: Now, "The alloys Hai use in their ships are way beyond our technology. They're twice as strong with half the weight. No wonder their ships are so fast."

File: data/hai/hai news.txt (hunk 17 above)
Comment (NomadicVolcano) [reply data/hai/hai news.txt]: Deleted.

The kid from Allhome that ended up in a box wasn't the only one. It is to set up lore for the Philosopher's Paradise. I see how this one looks bizarre and out-of-place though.

File: data/hai/hai news.txt (hunk 14 above)
Comment (NomadicVolcano) [reply data/hai/hai news.txt]: Deleted.

The kid from Allhome that ended up in a box wasn't the only one. It is to set up lore for the Philosopher's Paradise. I see how this one looks bizarre and out-of-place though.

File: data/hai/hai news.txt (hunk 13 above)
Comment (NomadicVolcano) [reply data/hai/hai news.txt:194]: Why not? There's plenty of merchants to transfer cargo and a lot of opportunity for profit. It is risky to go in person to make the deal, but it helps in negotiations. 

There are already several business-related stories and many story-related jobs; and the Hai are an adventurous species. There's no reason to believe Captain `<last>` is the only one getting those spaceport missions. The Captain probably has only talked to a few thousand Hai in a population that is at least in the ones of billions. Statistically speaking, of Captain `<last>` gets, say, one per year, then there are *at least* hundreds such missions per year.

File: data/hai/hai news.txt (hunk 10 above)
Comment (NomadicVolcano) [reply data/hai/hai news.txt]: Done.

File: data/hai/hai news.txt (hunk 12 above)
Comment (NomadicVolcano) [reply data/hai/hai news.txt:192]: This is to set up lore for the Philosopher's Paradise story. The excuse for the loss is intentionally unbelievable; the prison cruise was a trick and the vacationers were kidnapped.

File: data/hai/hai news.txt (hunk 13 above)
Comment (NomadicVolcano) [reply data/hai/hai news.txt:193]: This is to set up lore for the Philosopher's Paradise story which has the player rescue a Hai that was tricked into going to a human planet for business with a non-existent businessman.

File: data/hai/hai news.txt (hunk 2 above)
Comment (NomadicVolcano) [reply data/hai/hai news.txt:28]: "Back home, my favorite snack was pickled walnuts. Here, the pickled acorns are amazing!"

File: data/hai/hai news.txt (hunk 12 above)
Comment (NomadicVolcano) [reply data/hai/hai news.txt:193]: A fine point. I did all of this just to set up the lore for the Philosopher's Paradise story. Arachi was concerned that there weren't even rumors about people going missing. Adding such rumors belongs in that PR, not this one.

Comment (NomadicVolcano) [reply data/hai/hai news.txt]: I removed this news story. I'll keep it in the Philosopher's Paradise PR.

File: data/hai/hai news.txt (hunk 13 above)
Comment (NomadicVolcano) [reply data/hai/hai news.txt]: I removed this news story. I'll keep it in the Philosopher's Paradise PR.

**Comment by NomadicVolcano** [comment]
The branch is ready for review again. I think I made all of @Amazinite's requested changes, I merged the master, and all checks have passed.

**Pull request merged**